tests/test_accessibility.py

```diff
@@ -0,0 +1,26 @@
+"""
+Test accessibility auditing.
+
+You can run under phantomjs like this:
+SELENIUM_BROWSER=phantomjs ./run_tests.sh test_accessibility.py
+"""
+from bok_choy.web_app_test import WebAppTest
+from .pages import VisiblePage
+
+
+class AccessibilityTest(WebAppTest):
```

Comment:
Travis runs the tests only under firefox. Thus implicitly the test_axs_audit testcase will assert on the exception. And the successful audit is not actually performed under travis, which is why I put in that instruction on how to run under phantom locally.

You're right that it would be better if all this was explicit. And if travis runs the test under phantom.

I tried running all the tests under phantom and some fail. I feel that fixing them is outside the scope of this PR. I'll look into adding another commit which will run selected (maybe just this one) tests under phantom as well as the entire suite under firefox.